docker-compose-bridge.yml.template

```diff
@@ -18,6 +18,7 @@ services:
       - TZ=<timezone>
       - PLEX_CLAIM=<claimToken>
       - ADVERTISE_IP=http://<hostIPAddress>:32400/
+      - ALLOWED_NETWORKS=<hostIPAddressCIDRRange>
```

Comment:
This has no connection with `customConnections` nor does it affect the ability to use it.  This is the pref `allowedNetworks` which has the description `List of IP addresses and networks that are allowed without auth`.

This is no way we are going to ever set a default value for `allowedNetworks`, EVER!  It introduces a security risk and even if it's just to the host, that's still unacceptable.

Reply:
Hi, thanks for taking a look at the PR :)
My suggestion to add the `ALLOWED_NETWORKS` var in the docker-compose template comes from the fact that without allowing the local network it is not possible for the client to connect to the server without auth.
Maybe a comment can be added to the file, recommending to set the variable. I believe it can throw off many people using plex without auth otherwise

Reply:
> recommending to set the variable

We will not recommend setting this variable because of the security vulnerability.  Furthermore we will not recommend using Plex without authentication by extension of the same.  We have seen too many instances where people have exposed their server without realizing that they had done it via this variable (and another that we since obliterated).

Reply:
> without allowing the local network it is not possible for the client to connect to the server without auth.

The recommended solution to this problem is, of course, to authenticate your clients. The allowed-networks pref is a legacy feature and its use isn't recommended to new users.

Reply:
I understand it can be a source of trouble if misused, I just believe the user should be presented with the full picture.

Use this template as it is -> plex authentication required.
OR
add the variable at your own risk, according to your network setup -> authentication not required to use plex.

Better to warn the user yourself instead of letting them dig in the preferences and figure out a solution without being aware of the consequences :sweat_smile: